### ISSUE TITLE
Test failure: Interop/SimpleStruct/SimpleStruct/SimpleStruct.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#36969]
failed in job: [runtime-coreclr jitstress2-jitstressregs 20200524.1 ](https://dev.azure.com/dnceng/public/_build/results?buildId=658473&view=ms.vss-test-web.build-test-results-tab&runId=20434560&resultId=101478&paneView=debug)

Error message
~~~
Return code: 1
Raw output file: /home/helixbot/work/ADAE091F/w/A17C0967/e/Interop/SimpleStruct/Reports/Interop.SimpleStruct/SimpleStruct/SimpleStruct.output.txt
Raw output:
BEGIN EXECUTION
/home/helixbot/work/ADAE091F/p/corerun SimpleStruct.dll ''
Beginning scenario: Test #1 (Roundtrip of a simple structre by reference. Verify that values updated on unmanaged side reflect on managed side)
 Case 2: Direct p/invoke cdecl calling convention
 Case 4: Delegate p/invoke - cdecl calling convention
Beginning scenario: 

Test #2 (Roundtrip of a simple structre by value. Verify that values updated on unmanaged side reflect on managed side)
 Case 2: Direct p/invoke cdecl calling convention

Expected values:
 SimpleStruct->a=101
SimpleStruct->b=TRUE
SimpleStruct->c=10.11


Actual values:
 SimpleStruct->a=101
SimpleStruct->b=True
SimpleStruct->c=0

ERROR!!!-02: PInvokeTests->PosTest2 : Returned values are different from expected values
 Case 4: Delegate p/invoke cdecl calling convention

Expected values:
 SimpleStruct->a=101
SimpleStruct->b=TRUE
SimpleStruct->c=10.11


Actual values:
 SimpleStruct->a=101
SimpleStruct->b=True
SimpleStruct->c=3.142

ERROR!!!-02: PInvokeTests->PosTest2 : Returned values are different from expected values
FAIL
Expected: 100
Actual: 101
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/home/helixbot/work/ADAE091F/p
> /home/helixbot/work/ADAE091F/w/A17C0967/e/Interop/SimpleStruct/SimpleStruct/SimpleStruct.sh
Expected: True
Actual: False


Stack trace
   at Interop_SimpleStruct._SimpleStruct_SimpleStruct_._SimpleStruct_SimpleStruct_sh() in /__w/1/s/artifacts/tests/coreclr/Linux.x64.Checked/TestWrappers/Interop.SimpleStruct/Interop.SimpleStruct.XUnitWrapper.cs:line 140
~~~

**Comment by BruceForstall** [comment]
Failure mode: `CoreCLR Linux x64 Checked jitstress2_jitstressregs8 @ Ubuntu.1804.Amd64.Open`

@dotnet/jit-contrib

**Comment by CarolEidt** [comment]
Fixed by #36994 or #37062
